### ISSUE TITLE
Setting mediaType suffix to "" leaves dot

### ISSUE THREAD

**Issue by hatched** [gohugoio/hugo#3471]
In upgrading from Hugo 0.17 to 0.20.7 I had to make a few modifications to the RSS generation. The RSS on my site resides at `/rss`.

Thanks to https://discuss.gohugo.io/t/how-can-i-change-the-rss-url/118/17 I was able to find the missing fields in my toml to get it to properly generate only a single `rss.xml` file. However when I try to set the suffix to "" the generated file is `rss.` 

```TOML
[outputs]
  home = [ "RSS"]

[outputFormats]
[outputFormats.RSS]
  mediatype = "application/rss"
  baseName = "rss"

[mediaTypes]
[mediaTypes."application/rss"]
  suffix = ""
```

**Comment by bep** [comment]
That is by design, and for the RSS it may have been a breaking change since 0.17. We should print an ERROR about this, at least, that this isn't supported, but why is keeping the URL at `/rss`? important? It sounds a little odd to not wanting to indicate the media type in the URL.

**Comment by hatched** [comment]
For many years the rss for my site has been located at `/rss` so if I were to change it to `rss.xml` I'd  have to setup a redirect, which wasn't necessary before because I could set the `rssuri` to just `rss`. 

Using post aliases and renaming of `rssuri` gave an elegant no-server-changes-required way to port the blog over which also makes it easily portable between hosts now. 

I agree it's a little odd to set the suffix of all rss files to "" however I wasn't able to find another way to rename just the rss file (without a post-build `mv`). Is there a better way?

**Comment by Astorsoft** [comment]
As another issue linked to the trailing "." is the netlify "_redirects" and "_headers", I tried to follow the tutorial here: https://gohugo.io/news/http2-server-push-in-hugo/

First it was missing the mediatype definition for text/netlify
Second it was adding a trailing . even if I put suffix = ""

It generates _headers. and _redirects.

**Comment by bep** [comment]
OK, I forgot to close this, but this is fixed in Hugo >= 0.25.

You can set 

```
[mediaTypes]
[mediaTypes."application/rss"]
  suffix = ""
  delimiter = ""
```

Which should get yu what you want.
